test/validators.js

```diff
@@ -3419,6 +3419,28 @@ describe('Validators', function () {
         ],
       },
       {
+        locale: 'et-EE',
+        valid: [
+          '+372 512 34 567',
+          '372 512 34 4567',
+          '+37251234567',
+          '51234567',
+          '81234567',
+          '+372842345678',
+        ],
+        invalid: [
+          '12345',
+          '',
+          'NotANumber',
+          '+333 51234567',
+          '61234567',
+          '+51234567',
+          '+372 539 57 4',
+          '+372 900 1234',
+          '12345678'
```

Comment:
Missing a trailing comma here.

Reply:
Oh god, this is the last time I use the GitHub web editor. Keep making mistakes with it. Thank you!